lib/editor/actions/active.js

```diff
@@ -190,6 +191,59 @@ function updateUrl (
   }
 }
 
+/**
+ * Fetches a list of routes from the backend for the editor.
+ * Requires a lock to function.
+ */
+export function fetchRouteEntities (feedSource: Feed) {
+  return async function (dispatch: dispatchFn, getState: getStateFn) {
+    const state = getState()
+    const { lock } = state.editor.data
+    let lockCreated = false
+    // We need to get a lock if we do not already have one
+    if (!lock.sessionId) {
+      await dispatch(lockEditorFeedSource(feedSource.id))
+      // Save that lock was created so it can be removed when we are done.
+      lockCreated = true
+    }
+
+    const namespace = getEditorNamespace(
+      getState().editor.data.lock.feedId,
```

Comment:
We already have the state object from above. Could we just re-use that object (on the line below as well)?